Source/Scene/Implicit3DTileContent.js

```diff
@@ -164,22 +166,23 @@ Object.defineProperties(Implicit3DTileContent.prototype, {
  * up a promise chain to expand the immediate subtree.
  *
  * @param {Implicit3DTileContent} content The implicit content
+ * @param {Object} json The JSON containing the subtree. If parsing a binary subtree file, this will be undefined.
```

Comment:
also update the documentation in this file to mark `json` and `arrayBuffer` as mutually exclusive. Though the check only need to go in the constructor.